### PR TITLE
Render courts and HMCTS tribunals in this app

### PR DESCRIPTION
**Pull request by sihugh**
Courts and HMCTS tribunals are those pages accessed via https://www.gov.uk/courts-tribunals.  They're still rendered by Whitehall at present.

Rather than try to render these from the same view as organisation pages, I've split it out as there are some differences, and @oscarwyatt commented that it was getting hard to understand it all.

We don't show latest documents, as there are specialist finders set up for things like Employment Tribunal decisions.  These pages link to those where appropriate.

We don't show the FOI section, as this is the current state on Whitehall.

We don't show the corporate information section as this is the current state on Whitehall.

We don't show the "Read more" link at the bottom of the "What we do" section because we render the content from the about page in this section already. Showing the link would just link to the content that the user has just read, which would seem unhelpful.

I'm using the same style of test as the [taxon pages](https://github.com/alphagov/collections/blob/master/test/integration/taxon_browsing_test.rb).
I've put them in a separate test file from the other org pages, partly because the tests are in a different style.

I propose to move the org pages tests to this style at some point too.

https://trello.com/c/IRoHCQ3N/143-render-courts-and-tribunals-pages-from-collections

### CODE REVIEW

**Comment by sihugh** [comment]
This needs https://github.com/alphagov/govuk-content-schemas/pull/818 to be merged before tests will run

**Comment by sihugh** [comment]
These are the current pages:

https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/administrative-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/admiralty-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-business-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-business-and-property-courts
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/chancery-division-of-the-high-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/commercial-circuit-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/commercial-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/companies-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-competition-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/court-of-appeal-civil-division
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/court-of-appeal-criminal-division
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/court-of-protection
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/family-division-of-the-high-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-financial-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/insolvency-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/intellectual-property-enterprise-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-intellectual-property-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/northampton-county-court-business-centre
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/patents-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/planning-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-property-trusts-and-probate-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/queens-bench-division-of-the-high-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/the-revenue-list
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/senior-courts-costs-office
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/technology-and-construction-court
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/employment-appeal-tribunal
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/employment-tribunal
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-asylum-support
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-care-standards
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-criminal-injuries-compensation
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-general-regulatory-chamber
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-immigration-and-asylum
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-mental-health
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-property-chamber
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-social-security-and-child-support
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-special-educational-needs-and-disability
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-tax
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/first-tier-tribunal-war-pensions-and-armed-forces-compensation
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/gangmasters-licensing-appeals
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/reserve-forces-appeal-tribunal
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/upper-tribunal-administrative-appeals-chamber
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/upper-tribunal-immigration-and-asylum-chamber
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/upper-tribunal-lands-chamber
https://govuk-collections-pr-895.herokuapp.com/courts-tribunals/upper-tribunal-tax-and-chancery-chamber

File: app/views/organisations/_show_organisation.html.erb [hunk 1]
```diff
@@ -5,24 +5,28 @@
...
 <% end %>
 
 <% if @organisation.body || what_we_do.has_share_links? %>
   <%= render partial: 'what_we_do' %>
 <% end %>
 
-<% if @organisation.is_promotional_org? && @documents.has_promotional_features? %>
-  <%= render partial: 'promotional_features' %>
-<% end %>
+<% unless @organisation.is_court_or_hmcts_tribunal? %>
```
[the remark this answers is not in the document]
Comment (sihugh) [reply app/views/organisations/_show_organisation.html.erb]: I've split out courts and tribunals into a separate action/view to reduce the need for a bunch of guard clauses.  
I've also moved some of the other guard clauses inside partials which hopefully makes it easier to grok.

File: app/views/organisations/_header.html.erb [hunk 2]
```diff
@@ -14,6 +14,11 @@
           <%= t('organisations.part_of') %>
           <%= @show.parent_organisations %>
         </div>
+      <% elsif @organisation.is_court_or_hmcts_tribunal? %>
+        <div class="organisation__parent-organisations brand--<%= @organisation.brand %>">
```
Comment (andysellick) [inline app/views/organisations/_header.html.erb]: Do courts ever have a brand? If not we can remove `brand--<%= @organisation.brand %>` here.

**Comment by sihugh** [comment]
@tijmenb Thanks for taking a look!  I've added a `the` to the front of the courts and HMCTS tribunals now.  There's a [list of words](https://github.com/alphagov/collections/blob/90a0c18491bb202f9ff232685725339349f0420f/app/presenters/organisations/show_presenter.rb#L84-L108) which trigger this addition which includes `tribunal`.

I'll get Mia to look at the title, that's a good shout.  We're just inheriting what an org without a logo would get, but it might need a tweak.  Ganesh has had a look and has given a general 👍

**Comment by sihugh** [comment]
I'm going to merge this for now.  Mia'd like to change the title a bit, so we'll do that, but separately to minimise the complexity of reviewing all this stuff all over again.  
Courts and HMCTS tribunals are still rendered on Whitehall at the moment.  We'll do another product review before swapping the routes over.

**Pull request merged**